### PR TITLE
add the job creation permission for jobflow controller

### PR DESCRIPTION
**Pull request by william-wang**
fix issue #2987

### CODE REVIEW

**Comment by lowang-bh** [comment]
Need to do modification in `installer/helm/chart/volcano/templates/controllers.yaml` and then run `make update-development-yaml`

File: installer/volcano-development.yaml [hunk 1]
```diff
@@ -8527,7 +8527,7 @@ rules:
     verbs: ["create", "get", "list", "watch", "delete"]
   - apiGroups: ["batch.volcano.sh"]
     resources: ["jobs"]
-    verbs: ["get", "list", "watch", "update", "delete"]
+    verbs: ["create", "get", "list", "watch", "update", "delete"]
```
Comment (Thor-wl) [inline installer/volcano-development.yaml:8530]: Perhaps both helm chart and yaml file should been modified.

Comment (Thor-wl) [reply installer/volcano-development.yaml:8530]: Besides, maybe this fix should also been included in the latest three releases.

**Comment by william-wang** [comment]
> Need to do modification in `installer/helm/chart/volcano/templates/controllers.yaml` and then run `make update-development-yaml`

Thanks for reviewing. The pr is updated.

File: installer/volcano-development.yaml (hunk 1 above)
Comment (william-wang) [reply installer/volcano-development.yaml:8530]: The pr will be merged to release-1.8 as well.  The release-1.8 is the first release branch that includes jobflow.

Comment (Thor-wl) [reply installer/volcano-development.yaml:8530]: > The pr will be merged to release-1.8 as well. The release-1.8 is the first release branch that includes jobflow.

OK, got it.

**Review by Thor-wl — APPROVED** [review]

**Pull request merged**